pt/lessons/basics/modules.md

```diff
@@ -207,18 +207,69 @@ Se tentar chamar um macro que ainda não está carregado, Elixir vai gerar um er
 
 ### `use`
 
-Usa o módulo no contexto atual. Isto é particularmente útil quando um modulo que precisa executar alguma configuração. Ao chamar `use` nós invocamos o `__using__` dentro do modulo, proporcionando o módulo uma oportunidade para modificar nosso contexto existente:
+Com a macro `use` podemos dar habilidade a outro module para modificar a sua definição atual.
```

Comment:
I would change "module" to "módulo" in this sentence. Also, there are a few other "modulo"'s missing the accent in the next few lines.